src/app/rpmostree-builtin-compose.c

```diff
@@ -41,6 +41,9 @@ static RpmOstreeCommand compose_subcommands[] = {
   { "commit", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD | RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT,
     "Commit a target path to an OSTree repository",
     rpmostree_compose_builtin_commit },
+  { "commit2jigdo", RPM_OSTREE_BUILTIN_FLAG_LOCAL_CMD | RPM_OSTREE_BUILTIN_FLAG_REQUIRES_ROOT,
+    "Convert an OSTree commit into an OIRPM (jigdo)",
```

Comment:
Minor: maybe let's just stick with `OIRPM` as the final term and keep `jigdo` in e.g. documentation material? E.g. just

```
rpm-ostree compose oirpm
```

?

Reply:
I'd been thinking of the "whole thing" as "rpm-ostree jigdo", and "oirpm" is the special additional RPM.  But that said I'm totally open to changing this.